core/src/main/java/org/trimou/engine/config/SecurityActions.java

```diff
@@ -67,12 +67,7 @@ static ClassLoader getClassLoader(final Class<?> clazz) {
             return clazz.getClassLoader();
         }
         return AccessController
-                .doPrivileged(new PrivilegedAction<ClassLoader>() {
-                    @Override
-                    public ClassLoader run() {
-                        return clazz.getClassLoader();
-                    }
-                });
+                .doPrivileged((PrivilegedAction<ClassLoader>) clazz::getClassLoader);
```

Comment:
I do not like the usage of method references for `PrivilegedAction` stuff... esp. the need for casting.